tokio/src/net/unix/pipe.rs

```diff
@@ -16,6 +16,33 @@ cfg_io_util! {
     use bytes::BufMut;
 }
 
+/// Creates a new anonymous Unix pipe.
+///
+/// This function will open a new pipe and associate both pipe ends with the default
+/// event loop.
+///
+/// If you need to create a pipe for communicating with a spawned process, you can
+/// also see how to use `Stdio::piped()` with [`tokio::process`].
+///
+/// [`tokio::process`]: crate::process
+///
+/// # Errors
+///
+/// If creating a pipe fails, this function will return with the related OS error.
+///
+/// # Panics
+///
+/// This function panics if it is not called from within a runtime with
+/// IO enabled.
+///
+/// The runtime is usually set implicitly when this function is called
+/// from a future driven by a tokio runtime, otherwise runtime can be set
+/// explicitly with [`Runtime::enter`](crate::runtime::Runtime::enter) function.
+pub fn new() -> io::Result<(Sender, Receiver)> {
```

Comment:
I'd prefer to call this something else. We have similar methods for various message passing channels, and there the method is called "channel".
```suggestion
pub fn pipe() -> io::Result<(Sender, Receiver)> {
```

Reply:
I agree, `pipe` sounds more reasonable.